Application/EdFi.Ods.Api.Startup/Features/Installers/CoreApiInstaller.cs

```diff
@@ -235,22 +242,26 @@ private void RegisterUniqueIdToUsiValueMapper(IWindsorContainer container)
 
         private void RegisterPersonUniqueIdToUsiCache(IWindsorContainer container)
         {
-            container.Register(
-                Component
-                    .For<IPersonUniqueIdToUsiCache>()
-                    .ImplementedBy<PersonUniqueIdToUsiCache>()
-                    .DependsOn(Dependency.OnValue("synchronousInitialization", false)));
+            container.Register(Component.For<IPersonUniqueIdToUsiCache>()
+                .ImplementedBy<PersonUniqueIdToUsiCache>()
+                .DependsOn(Dependency.OnValue("slidingExpiration",
```

Comment:
Since we can't actually do sliding expiration with our current cache implementations, this should be removed everywhere.

Reply:
As per standup, this is still using the Asp.Net cache, so it correctly handles expirations.